### PR TITLE
chore(docs): document new types in count_distinct()

### PR DESCRIPTION
**Pull request by puzpuzpuz**
Core PR counterpart: https://github.com/questdb/questdb/pull/2439

### CODE REVIEW

File: docs/reference/function/aggregation.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ SELECT payment_type, count_distinct(counterparty) FROM transactions;
 
 :::note
 
-`null` values are not counted in `count_distinct` functions.
+`null` values are not counted in `count_distinct` function.
```
Comment (amyshwang) [inline docs/reference/function/aggregation.md:122]:
```suggestion
`null` values are not counted in the `count_distinct` function.
```

File: docs/reference/function/aggregation.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ SELECT payment_type, count() FROM transactions;
 ## count_distinct
 
 `count_distinct(STRING_COL)` or `count_distinct(SYMBOL_COL)` - counts distinct
```
Comment (amyshwang) [inline docs/reference/function/aggregation.md:87]: Maybe the input needs to be changed, too?

```suggestion
`count_distinct(column)` - counts distinct
```

**Review by amyshwang — CHANGES_REQUESTED** [review]

File: docs/reference/function/aggregation.md (hunk 1 above)
Comment (puzpuzpuz) [reply docs/reference/function/aggregation.md]: Addressed in 73d279e61665e119f1de06be5e2229e354ac0a95

File: docs/reference/function/aggregation.md (hunk 2 above)
Comment (puzpuzpuz) [reply docs/reference/function/aggregation.md]: Addressed in 73d279e61665e119f1de06be5e2229e354ac0a95

**Review by amyshwang — APPROVED** [review]
Looks good!

**Pull request merged**